requirements.txt

```diff
@@ -2,3 +2,5 @@ simplejson
 pyserial
 pyyaml
 coveralls
+lxml
+click
```

Comment:
We have installed click from install_nightly script when is required.
Then we dont need it here.

IMHO here should be just odoo requirements